app/services/permissions/order.rb

```diff
@@ -28,27 +36,35 @@ def visible_line_items
 
     # Any line items that I can edit
     def editable_line_items
-      Spree::LineItem.where(order_id: editable_orders)
+      Spree::LineItem.where(order_id: editable_orders.select(:id))
     end
 
     private
 
     # Any orders placed through any hub that I manage
-    def managed_orders
-      Spree::Order.where(distributor_id: @permissions.managed_enterprises.select("enterprises.id"))
+    def managed_orders_where_values
+      Spree::Order.
+        where(distributor_id: @permissions.managed_enterprises.select("enterprises.id")).
+        where_values.
+        reduce(:and)
     end
 
     # Any order that is placed through an order cycle one of my managed enterprises coordinates
-    def coordinated_orders
-      Spree::Order.where(order_cycle_id: @permissions.coordinated_order_cycles.select(:id))
+    def coordinated_orders_where_values
+      Spree::Order.
+        where(order_cycle_id: @permissions.coordinated_order_cycles.select(:id)).
```

Comment:
Semaphore is complaining about an ambiguous `id` reference. Maybe this needs to be `"DISTINCT spree_orders.id"` as well?

Reply:
yeah, it's in order_and_distributors report, fixed in new commit.